elitzur-avro/src/main/scala/com/spotify/elitzur/converters/avro/dynamic/dsl/Implicits.scala

```diff
@@ -0,0 +1,45 @@
+package com.spotify.elitzur.converters.avro.dynamic.dsl
+
+import com.spotify.elitzur.converters.avro.dynamic.{
+  ArrayValidatorOp,
+  OptionValidatorOp,
+  ValidatorOp
+}
+
+import scala.annotation.tailrec
+
+object Implicits {
+  implicit class AccessorOps(val accessors: List[BaseAccessor]) {
```

Comment:
Does this need to be an implicit class? I would rather have `combineFns` and `toValidatorOp` take a list of accessors explicitly. I feel like it was hard to tell where these were coming from above.